source/ui/data/VerticalDelegate.js

```diff
@@ -214,18 +233,33 @@
 		* @private
 		*/
 		childSize: function (list) {
-			if (!list.fixedChildSize) {
-				var pageIndex = list.$.page1.index,
+			if (!list.fixedChildSize || !list.childSize) {
+				var page = list.$.page1,
 					sizeProp  = list.psizeProp,
 					n         = list.$.page1.node || list.$.page1.hasNode(),
```

Comment:
For example, if there is only a single model in collection?
Currently, {{updatePage}} request from generatePage is ignored inside of updatePage logic.
However, it is not proper code. Let me fix it.

Thank you for your review. :+1: